src/components/10-atoms/input-text/index.js

```diff
@@ -104,15 +173,30 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       // yes, set UI from model state
       this.nativeInput.value = this.modelValue;
     }
+
+    if (this.maxLength) {
+      this.modelCounter = this.getCounterText;
+    }
   };
 
+  isNotAtMaxLength() {
```

Comment:
I doubt this is needed. If maxLength could ever be of type `string`, just use `==` instead of `===`.

Reply:
Uh, are we allowing that? If yes, I will go for it.

Reply:
Eslint forbids it.

Reply:
I wanted to fix it, but `nativeValue` is sometimes `string` and sometimes `number`.